playbooks/rpm_packaging.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: all
+  tasks:
+    - name: Enable Fedora Copr for gem-compair
+      command: "dnf copr enable -y ekohl/gem-compare"
+      args:
+        creates: /etc/yum.repos.d/_copr_ekohl-gem-compare.repo
+      become: true
+
+    - name: Install dependencies for packaging tooling
+      yum:
+        name: "{{ package_list }}"
+        state: present
+      vars:
+        package_list:
+          - git-core
+          - git-annex
+          - rubygem-nokogiri
+          - rubygem-gem-compare
```

Comment:
Looks like I locally depend on an ancient package still installed from Fedora 24 and COPR doesn't have repoclosure so it currently fails.

Reply:
Yes. The question is, should we update the rpm, or drop the top commit from this PR?

Reply:
I'd also be ok changing the bumping script to not hard depend on gem-compare. Its primary benefit to me has been showing differences in the requirements. When we automatically update the dependencies, the biggest benefit is seeing if files were added/removed. The build will fail in most cases.